backend/dal/Services/ResearchFileService.cs

```diff
@@ -85,5 +83,20 @@ public Paged<PimsResearchFile> GetPage(ResearchFilter filter)
             _user.ThrowIfNotAuthorized(Permissions.ResearchFileView);
             return _researchFileRepository.GetPage(filter);
         }
+
+        private void PopulateResearchFile(PimsProperty property)
+        {
+            property.PropertyClassificationTypeCode = "CORESTRAT"; // Todo: should be "UNKNOWN"
+            property.PropertyDataSourceEffectiveDate = System.DateTime.Now;
+            property.PropertyDataSourceTypeCode = "PAIMS"; // Todo: should be "PMBC"
+
+            property.PropertyTypeCode = "UNKNOWN";
+
+            property.PropertyStatusTypeCode = "UNSURVYED"; // Todo: should be 'UNKNOWN';
```

Comment:
are these todos being tracked? it seems like we need db changes for the type codes and for the region/district we need a story to get those from the map.